bin/kramdown

```diff
@@ -46,6 +42,9 @@ OptionParser.new do |opts|
           "html, GFM or markdown") {|v| options[:input] = v}
   opts.on("-o", "--output ARG", Array, "Specify one or more output formats separated by commas: " \
           "html (default),", "kramdown, latex, pdf, man or remove_html_tags") {|v| format = v}
+  opts.on("-n", "--no-rcfile", "Do not read any configuration file") {config_file = nil}
+  opts.on("-r", "--rcfile FILE", "Read specified configuration file, if it exists.",
+          "Default: #{config_file}") {|v| config_file = v}
 
```

Comment:
I would go with `--config-file FILE` and `--no-config-file`, no short codes. And mention in `--no-config-file` that the default is using it if it is available. Should also go into the man page.